source/androidx.viewpager2/viewpager2/transforms/Metadata.xml

```diff
@@ -0,0 +1,18 @@
+﻿<metadata>
+    <attr
+        path="/api/package[@name='androidx.viewpager2.adapter']/class[@name='FragmentStateAdapter']/method[@name='onCreateViewHolder' and count(parameter)=2 and parameter[1][@type='android.view.ViewGroup'] and parameter[2][@type='int']]"
+        name="final"
+        >
+        false
+    </attr>
+    <attr
+        path="/api/package[@name='androidx.viewpager2.adapter']/class[@name='FragmentStateAdapter']/method[@name='onCreateViewHolder' and count(parameter)=2 and parameter[1][@type='android.view.ViewGroup'] and parameter[2][@type='int']]"
+        name="managedReturn"
+        >
+        Java.Lang.Object
+    </attr>
+    <remove-node
+        path="/api/package[@name='androidx.viewpager2.adapter']/class[@name='FragmentStateAdapter']/method[@name='setHasStableIds' and count(parameter)=1 and parameter[1][@type='boolean']]"
```

Comment:
Why is this removed?

Reply:
Can you try to build locally and see what is generated in RecyclerView for this method?
Weird.

It should be in 
```
./generated/androidx.recyclerview.recyclerview/obj/Release/monoandroid90/generated/src/AndroidX.RecyclerView.Widget.RecyclerView.cs:124
```

Reply:
Will try `managedName` or `propertyName` as you suggested.

Reply:
Fixed by fixes in RecyclerView and then ViewPager2